griptape/tasks/image_inpainting_task.py

```diff
@@ -0,0 +1,119 @@
+from __future__ import annotations
+
+import os
+from os import path
+from typing import Optional
+
+from attr import define, field
+
+from griptape.artifacts import ImageArtifact
+from griptape.engines.image.image_generation_engine import ImageGenerationEngine
+from griptape.loaders.image_loader import ImageLoader
+from griptape.rules import Rule, Ruleset
+from griptape.tasks import BaseTextInputTask
+
+
+@define
+class ImageInpaintingTask(BaseTextInputTask):
```

Comment:
So this is interesting, the Tasks's input is text _and_ image. I think we want users to be able to pass the `ImageArtifact`s with something similar to what was implemented [here](https://github.com/griptape-ai/griptape/pull/494). I just don't love the idea of always having to serialize the images in order to pass them between Tasks.

Maybe the input should be a tuple of `ImageArtifact` + `TextArtifact`?